github-workflow/scripts/create-issue.sh

```
@@ -0,0 +1,90 @@
+#   SIZE           - Size (XS/S/M/L/XL)
+#   STATUS         - Status (기본: Todo)
+#   ESTIMATE       - Estimate 값
+
+set -e
+
+TITLE="${1:-}"
+[ -z "$TITLE" ] && echo "ERROR: 제목을 지정해야 합니다." >&2 && exit 1
+
+OWNER=$(gh repo view --json owner --jq '.owner.login')
+CURRENT_REPO=$(gh repo view --json name --jq '.name')
+TARGET_REPO="${EPIC_REPO:-$CURRENT_REPO}"
```

Shellcheck (0.11.0)

[warning] TARGET_REPO appears unused. Verify use (or export if used externally).

(SC2034)